docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>Maybe say that the function swaps the width and height of images, leaving the channels in the shape in place.</p><p> </p><p>I would also add to this paragraph a title, something like "Defining the Processing Function"</p><p> </p><p>Optionally:</p><p>We also had the suggestions from our tech writer, how the tutorials should be written, an the suggested flow would be to use something similar to enumeration of steps.</p><p> </p><p>I think you can repeat before the functions you define:</p><p> </p><p>"Define the shape function swapping the width and height"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>